<p align="center">
    <a href="https://latch.bio/">
        <img src="images/biokit_logo.jpg" width=37.5%>
    </a>
    <a href="https://latch.bio/">
        <img src="images/latch_logo.png" width=50%>
    </a>
    <a href="https://console.latch.bio/explore/60054/info">
    <br/>
    <span style="font-size:larger;">Click here to see the workflow!</span></br>
    </a>
    </br></br>
    <a href="https://github.com/hannahle/alignment/graphs/contributors" alt="Contributors">
        <img src="https://img.shields.io/github/contributors/jlsteenwyk/latch_wf_clipkit">
    </a>
    <a href="https://twitter.com/intent/follow?screen_name=jlsteenwyk" alt="Author Twitter">
        <img src="https://img.shields.io/twitter/follow/jlsteenwyk?style=social&logo=twitter"
            alt="follow on Twitter">
    </a>
</p>

</br>

# Codon optimization with BioKIT
## About
Estimates of codon optimization can be used to identify
genes that are expressed at high levels and are important
for organismal ecology. Here, we will use the metric
gRSCU (gene-wise relative synonymous codon usage), which
is a function in the bioinformatic toolkit
[BioKIT](https://jlsteenwyk.com/BioKIT),
to estimate codon optimization.

<br/>

## gRSCU
gRSCU is calculated by determining the mean or median
relative synonymous codon usage value for all codons in each
gene based on their genome-wide values.

<br/>

## Citation
If you found [BioKIT](https://jlsteenwyk.com/BioKIT) useful,
please cite *BioKIT: a versatile toolkit for processing and
analyzing diverse types of sequence data*. Steenwyk et al.
2022, GENETICS. doi:
[10.1093/genetics/iyac079](https://academic.oup.com/genetics/advance-article-abstract/doi/10.1093/genetics/iyac079/6583183).

